dependencies/CMakeLists.txt

```diff
@@ -0,0 +1,224 @@
+cmake_minimum_required(VERSION 3.13.2)
```

Comment:
It is an independent project, with only a CMakeLists populated with FetchContent that downloads, and builds all the dependencies. It includes logic to handle all the supported platforms.
The nice thing about being a subproject, is that we only have to compile it once (maybe nightly, or weekly), and then other DGtal pipelines can download it from the azure servers. Right now is configured to compile nightly.

It is used because we also have to compile the dependencies to generate the python wheel. Right now only zlib and boost, but maybe in the future we need to add more for the python wheels.

Reply:
The minimum cmake version was thoughtless, I will change it to match the DGtal one.